packages/release/Cargo.toml

```diff
@@ -7,3 +7,26 @@ build = "build.rs"
 
 [lib]
 path = "pkg.rs"
+
+[build-dependencies]
+acpid = { path = "../acpid" }
+ca-certificates = { path = "../ca-certificates" }
+chrony = { path = "../chrony" }
+coreutils = { path = "../coreutils" }
+dbus-broker = { path = "../dbus-broker" }
+e2fsprogs = { path = "../e2fsprogs" }
+filesystem = { path = "../filesystem" }
+glibc = { path = "../glibc" }
+grub = { path = "../grub" }
+host-ctr = { path = "../host-ctr" }
+iproute = { path = "../iproute" }
+kernel = { path = "../kernel" }
```

Comment:
This can wait for the 5.10 kernel to come in, but we'd want to move this to the variant's dependencies rather than release, since it'll no longer be the same for all variants.

Reply:
Agreed. The release package has consistently struck me as an abstraction worth poking at/questioning as I've been getting more adept with the build. I have a feeling that, as variants become more specialized, the number of things in release may reduce over time.